app/furniture/marketplace/orders_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Marketplace
   class OrdersController < Controller
     expose :order, scope: -> { orders }, model: Order
-    expose :orders, -> { OrderPolicy::Scope.new(shopper, marketplace.orders).resolve.order(created_at: :desc) }
+    expose :orders, -> { policy_scope(marketplace.orders).order(created_at: :desc) }
```

Comment:
This is the entire point of this PR... 😭

Reply:
I know. 🤗